libnet/common.c

```diff
@@ -65,6 +65,41 @@ bool libnet_init(unsigned char *tags_to_register, unsigned tags_to_register_numb
 	return false;
 }
 
+ssize_t libnet_wait_for_tag(unsigned char tag, char *buffer, size_t length) {
```

Comment:
maybe change char to unsigned char because send sends unsigned char and tlv structure also has unsigned char